src/modelgauge/prompt_pipeline.py

```diff
@@ -41,17 +41,18 @@ def __len__(self):
 
 
 class CsvPromptInput(PromptInput):
-    def __init__(self, path):
+    def __init__(self, path, sut_options: Optional[SUTOptions] = None):
         super().__init__()
         self.path = path
+        self.sut_options = SUTOptions() if sut_options is None else sut_options
         self._validate_file()
 
     def __iter__(self) -> Iterable[PromptWithContext]:
         with open(self.path, newline="") as f:
             csvreader = csv.DictReader(f)
             for row in csvreader:
                 yield PromptWithContext(
-                    prompt=TextPrompt(text=row["Text"]),
+                    prompt=TextPrompt(text=row["Text"], options=self.sut_options),
```

Comment:
It seems a little weird to me to have to set this for each prompt given that it's stable across the run, but I guess that's the object model we have for now.

Reply:
Agreed